### PR TITLE
CDA -> FHIR transformation

### PR DESCRIPTION
**Pull request by teslajoy**
The goal is translate initial minimum CDA data into FHIR schema graph. 
this pull request should translate: 
- Patient 
- Condition 
- Specimen
- ResearchStudy 
- ResearchSubject 
- Observation w Specimen and Patient focus 
- BodyStructure 

 remaining: 
1. Some relations are still to be refined ex. dbgap to projects 
2. use-case testing 
3. Medication and Medication administration and genomics variants results as observations 

run on all data: 
```
cda2fhir transform 
```

current test takes ~5min : 
```
pytest -cov
```

### CODE REVIEW

**Comment by teslajoy** [comment]
closes #11 #12

File: cda2fhir/transformer.py [hunk 1]
```diff
@@ -0,0 +1,722 @@
...
+        elif sex in ['female', 'Female', 'F']:
+            return female
+        elif sex in ['Unspecified', 'Not specified in data', 'O', 'U', '0000']:  # clarify definitions
+            return unknown
+
+    @staticmethod
+    def map_ethnicity(ethnicity: str) -> Extension:
+        """map CDA ethnicity content to FHIR patient extension."""
+        url = "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity"
+        cda_ethnicity = [None, 'not hispanic or latino', 'hispanic or latino',
```
Comment (bwalsh) [inline cda2fhir/transformer.py]: Check pycharm's warnings:
image

File: tests/integration/test_transorm.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import os
```
Comment (bwalsh) [inline tests/integration/test_transorm.py:1]: typo: test_transorm.py sb: test_transform.py

File: README.md [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
+
+- example 
+``` 
+cda2fhir transform 
+``` 
+
+### Testing
+```
+pytest -cov 
+```
```
Comment (bwalsh) [inline README.md:48]: Add a "how to" run g3t validate, how long should it take, any expected warnings etc.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,43 @@
...
+## Usage 
 ### Installation
 
+- from source 
 ```commandline
 # clone repo & setup virtual env
 python3 -m venv venv
 . venv/bin/activate
 pip install -e .
 ```
```
Comment (bwalsh) [inline README.md:19]: * Add Related projects: section to point to cda aggregator project
* Add a "contact the author" to retrieve the raw data

**Review by bwalsh — APPROVED** [review]
Made a few inline comments.  Go ahead and merge when complete.

**Pull request merged**